File: Explore/ExploreStructures/VBStructures/Class1.vb
Public Class Class1

    Public Structure MyStruct1
        Public i As Int16
        Public j As Int32
        Public s1 As String
        Public s2 As String
    End Structure

    Public Structure MyStruct2
        Public i As Int16
        Public j As Int32
        <VBFixedString(10)> Public s1 As String
        <VBFixedString(20)> Public s2 As String
    End Structure

End Class

File: Explore/ExploreStructures/CSharpStructures/Class1.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;
using Microsoft.VisualBasic;

namespace CSharpStructures
{
    public class Class1
    {
        public struct MyStruct1
        {
            public Int16 i;
            public Int32 j;
            public string s1;
            public string s2;
        }

        public struct MyStruct2
        {
            public Int16 i;
            public Int32 j;
            [VBFixedString(10)]
            public string s1;
            [VBFixedString(20)]
            public string s2;
        }
    }
}
